### ISSUE TITLE
Expose Plotman's controller logs for plotting and archiving, remove redundant output

### ISSUE THREAD

**Issue by zorner** [guydavis/machinaris#811]
Thank you. Thank you. Thank you. The addition of the time stamps was very much need. It is a great improvement.

**Describe the request**

Remove the looped entries in plotman.log. They are cluttering up the log and have no purpose most of the time. They belong in a trace or debut log, but not the normal info log. The user wants to know what is happening.

I have seen strategies where entries like these are consolidated into one entry followed by "This repeats X number of times" until something ends the loop and then one entry is put in again as the loop restarts.

The big offenders look to be:
Start root logger
[archive] No free archive dirs found.
[archive] No plots found
[archive] pid: XXXX, XXXX, XXXX
[archive] Disk space check timed out in 5 seconds
[plot] stagger (XXXXs/1800s)
[plot] no eligible tempdirs (XXXXs/1800s)

**Logs:**

```
2022-11-12T18:49:11.922082-05:00: Start root logger
2022-11-12T18:49:11.934227-05:00: [archive] ...starting archive loop
2022-11-12T18:49:12.075320-05:00: [archive] No free archive dirs found.
2022-11-12T18:49:16.712727-05:00: Start root logger
2022-11-12T18:49:26.442729-05:00: Start root logger
2022-11-12T18:49:32.165310-05:00: [archive] No free archive dirs found.
2022-11-12T18:49:52.258597-05:00: [archive] No free archive dirs found.
2022-11-12T18:50:12.350171-05:00: [archive] No free archive dirs found.
2022-11-12T18:50:32.440880-05:00: [archive] No free archive dirs found.
2022-11-12T18:50:52.531057-05:00: [archive] No free archive dirs found.
2022-11-12T18:51:12.623139-05:00: [archive] No free archive dirs found.
2022-11-12T18:51:23.760896-05:00: Start root logger
2022-11-12T18:53:08.881743-05:00: Start root logger
2022-11-12T18:53:42.339769-05:00: Start root logger
2022-11-12T18:54:40.335657-05:00: Start root logger
2022-11-12T18:54:40.347666-05:00: [plot] Starting plot job: chia_plot -n 1 -k 32 -r 24 -u 256 -x 8444 -t /plotting3/ -d /plotting3/ -2 /ramdrive/ -v 256 -K 1 -f a3f7e719a508879c16144b615e76aeab1c2bea66b47b690aea33c9e5fed591ad4476b573f2e3dddfe84e923adeae6449 -c xch1ekslymyyfrrv50nptterxuxgmwvw0vc8xz848sf3gzfhse7e8thqmjlqj4 ; logging to /root/.chia/plotman/logs/2022-11-12T18_54_40.340165-05_00.plot.log
2022-11-12T18:55:00.373476-05:00: [plot] stagger (20s/1800s)
2022-11-12T18:55:20.399451-05:00: [plot] stagger (40s/1800s)
2022-11-12T18:55:40.421517-05:00: [plot] stagger (60s/1800s)
2022-11-12T18:55:46.622256-05:00: Start root logger
2022-11-12T18:55:54.126264-05:00: Start root logger
2022-11-12T18:56:00.453659-05:00: [plot] stagger (80s/1800s)
```

```
2022-11-12T19:30:21.948055-05:00: Start root logger
2022-11-12T19:30:30.678800-05:00: [plot] stagger (82s/1800s)
2022-11-12T19:30:41.241519-05:00: [archive] pid: 7032, 7033, 7034
2022-11-12T19:30:51.114683-05:00: [plot] stagger (103s/1800s)
2022-11-12T19:30:59.064404-05:00: Start root logger
2022-11-12T19:31:01.264776-05:00: [archive] pid: 7032, 7033, 7034
2022-11-12T19:31:11.505408-05:00: [plot] stagger (123s/1800s)
2022-11-12T19:31:21.270283-05:00: [archive] pid: 7032, 7033, 7034
2022-11-12T19:31:31.948197-05:00: [plot] stagger (144s/1800s)
2022-11-12T19:31:41.296199-05:00: [archive] pid: 7032, 7033, 7034
2022-11-12T19:31:52.422342-05:00: [plot] stagger (164s/1800s)
2022-11-12T19:32:01.321529-05:00: [archive] pid: 7032, 7033, 7034
2022-11-12T19:32:12.867637-05:00: [plot] stagger (185s/1800s)
2022-11-12T19:32:21.343481-05:00: [archive] pid: 7032, 7033, 7034
2022-11-12T19:32:33.358885-05:00: [plot] stagger (205s/1800s)
2022-11-12T19:32:41.353962-05:00: [archive] pid: 7032, 7033, 7034
2022-11-12T19:32:53.892245-05:00: [plot] stagger (226s/1800s)
```
```
2022-11-11T18:18:39.222582-05:00: [plot] stagger (1765s/1800s)
2022-11-11T18:18:59.435688-05:00: [plot] stagger (1785s/1800s)
2022-11-11T18:19:19.595157-05:00: [plot] no eligible tempdirs (1806s/1800s)
2022-11-11T18:19:39.845425-05:00: [plot] no eligible tempdirs (1826s/1800s)
2022-11-11T18:19:52.512364-05:00: Start root logger
2022-11-11T18:20:00.059273-05:00: [plot] no eligible tempdirs (1846s/1800s)
2022-11-11T18:20:20.224729-05:00: [plot] no eligible tempdirs (1866s/1800s)
2022-11-11T18:20:40.537576-05:00: [plot] no eligible tempdirs (1887s/1800s)
2022-11-11T18:21:00.709249-05:00: [plot] no eligible tempdirs (1907s/1800s)
2022-11-11T18:21:21.593827-05:00: [plot] no eligible tempdirs (1928s/1800s)
2022-11-11T18:21:41.811042-05:00: [plot] no eligible tempdirs (1948s/1800s)
2022-11-11T18:21:52.657342-05:00: Start root logger
2022-11-11T18:22:01.872727-05:00: [plot] no eligible tempdirs (1968s/1800s)
2022-11-11T18:22:22.184619-05:00: [plot] no eligible tempdirs (1988s/1800s)
2022-11-11T18:22:42.485961-05:00: [plot] no eligible tempdirs (2008s/1800s)
2022-11-11T18:23:02.809856-05:00: [plot] no eligible tempdirs (2029s/1800s)
2022-11-11T18:23:23.143588-05:00: [plot] no eligible tempdirs (2049s/1800s)
2022-11-11T18:23:43.483056-05:00: [plot] no eligible tempdirs (2069s/1800s)
2022-11-11T18:24:03.790039-05:00: [plot] no eligible tempdirs (2090s/1800s)
2022-11-11T18:24:23.940833-05:00: [plot] no eligible tempdirs (2110s/1800s)
2022-11-11T18:24:25.359421-05:00: Start root logger
2022-11-11T18:24:44.198195-05:00: [plot] no eligible tempdirs (2130s/1800s)
2022-11-11T18:25:04.487396-05:00: [plot] no eligible tempdirs (2150s/1800s)
2022-11-11T18:25:24.623261-05:00: [plot] no eligible tempdirs (2171s/1800s)
2022-11-11T18:25:44.748123-05:00: [plot] no eligible tempdirs (2191s/1800s)
2022-11-11T18:26:04.862697-05:00: [plot] no eligible tempdirs (2211s/1800s)
2022-11-11T18:26:25.007906-05:00: [plot] no eligible tempdirs (2231s/1800s)
2022-11-11T18:26:45.199139-05:00: [plot] no eligible tempdirs (2251s/1800s)
2022-11-11T18:27:05.409292-05:00: [plot] no eligible tempdirs (2271s/1800s)
2022-11-11T18:27:10.027389-05:00: Start root logger
2022-11-11T18:27:25.640273-05:00: [plot] no eligible tempdirs (2292s/1800s)
```

```
2022-11-13T06:20:53.295216-05:00: [archive] No plots found
2022-11-13T06:20:56.335503-05:00: [plot] stagger (1772s/1800s)
2022-11-13T06:21:13.332737-05:00: [archive] No plots found
2022-11-13T06:21:16.352766-05:00: [plot] stagger (1792s/1800s)
2022-11-13T06:21:33.375462-05:00: [archive] No plots found
2022-11-13T06:21:36.473993-05:00: [plot] Starting plot job: chia_plot -n 1 -k 32 -r 24 -u 256 -x 8444 -t /plotting3/ -d /plotting3/ -2 /ramdrive/ -v 256 -K 1 -f a3f7e719a508879c16144b615e76aeab1c2bea66b47b690aea33c9e5fed591ad4476b573f2e3dddfe84e923adeae6449 -c xch1ekslymyyfrrv50nptterxuxgmwvw0vc8xz848sf3gzfhse7e8thqmjlqj4 ; logging to /root/.chia/plotman/logs/2022-11-13T06_21_36.381620-05_00.plot.log
2022-11-13T06:21:53.420475-05:00: [archive] No plots found
2022-11-13T06:21:56.516497-05:00: [plot] stagger (20s/1800s)
2022-11-13T06:22:13.454426-05:00: [archive] No plots found
2022-11-13T06:22:16.549649-05:00: [plot] stagger (40s/1800s)
2022-11-13T06:22:27.291331-05:00: Start root logger
2022-11-13T06:22:33.505184-05:00: [archive] No plots found
2022-11-13T06:22:36.583941-05:00: [plot] stagger (60s/1800s)
2022-11-13T06:22:53.549730-05:00: [archive] No plots found
2022-11-13T06:22:56.632824-05:00: [plot] stagger (80s/1800s)
2022-11-13T06:23:13.617914-05:00: [archive] No plots found
2022-11-13T06:23:16.659205-05:00: [plot] stagger (100s/1800s)
```

```
2022-11-15T03:18:07.909994-05:00: [archive] Disk space check timed out in 5 seconds
2022-11-15T03:18:20.002316-05:00: Start root logger
2022-11-15T03:18:32.943892-05:00: [archive] Disk space check timed out in 5 seconds
2022-11-15T03:18:57.957771-05:00: [archive] Disk space check timed out in 5 seconds
2022-11-15T03:19:12.630493-05:00: Start root logger
2022-11-15T03:19:22.990894-05:00: [archive] Disk space check timed out in 5 seconds
2022-11-15T03:19:48.024336-05:00: [archive] Disk space check timed out in 5 seconds
2022-11-15T03:20:13.058103-05:00: [archive] Disk space check timed out in 5 seconds
```
Please note "Start root logger" dose not have a service name in the []. Also, "[archive] pid: 7032, 7033, 7034", I am guessing where archive is moving a plot. The plot name, source and destination would be more helpful.

**Comment by guydavis** [comment]
Good suggestions, I'll look to improve this.  FYI, I've just added a table showing the progress/result of each `plotman archive` result, whether local or remote rsync.  Example using the most recent `machinaris:develop` image where `merry` is my plotter machine and `aragorn` is my farmer/harvester machine.

![image](https://user-images.githubusercontent.com/12936/201539062-088aa9f5-a9dd-4604-a96f-7d6b047195a2.png)

**Comment by zorner** [comment]
Screen shot looks good and I can see the place for the archive data, but as I am using custom transcripts, the info does not populate.

**Comment by guydavis** [comment]
This is now in the `machinaris:test` images.  Will be in the [upcoming v0.8.6 release](https://github.com/guydavis/machinaris/wiki/Releases).

**Comment by guydavis** [comment]
Now available in version of [0.8.6](https://github.com/guydavis/machinaris/releases/tag/v0.8.6) of Machinaris. Thanks!
